sdcm/cluster_k8s/__init__.py

```diff
@@ -497,8 +505,11 @@ def deploy_scylla_manager(self, pool_name: str = None) -> None:
                     namespace=SCYLLA_MANAGER_NAMESPACE,
                 ))
 
-        self.kubectl("wait --timeout=10m --all --for=condition=Ready pod",
-                     namespace=SCYLLA_MANAGER_NAMESPACE)
+        self.kubectl_wait(
+            "--all --for=condition=Ready pod",
+            namespace=SCYLLA_MANAGER_NAMESPACE,
+            timeout=600,
+        )
```

Comment:
The same here. We had here "kubectl wait" and it is added as "kubectl wait". Don't see any need to update this code.